pyproject.toml

```diff
@@ -122,6 +122,7 @@ disable = [
 # NOTE: Do not create an exclude list. Edit .lintrunner.toml instead
 target-version = "py39"
 unsafe-fixes = true
+# "TC", # flake8-type-checking
```

Comment:
Do you intend to enable or remove this rule?

Reply:
Disable it because more files need to be changed.